docker-compose-dev.yml

```diff
@@ -14,7 +14,7 @@ services:
       DASH_DEBUG_MODE: "True"
       DASH_SILENCE_ROUTES_LOGGING: "False"
       DASH_SERVER_PORT: 8050
-      DB_HOST: db
+      DB_HOST: mongodb://db/openpath_prod_ca_ebike
```

Comment:
Was this change intentional? Shouldn't the default still be `Stage_database` ? @shankari @TeachMeTW

Reply:
Ah, I see this gets overwritten in the new version of the script so this value is arbitrary.

I still think the placeholder should be something other than `openpath_prod_ca_ebike` to prevent confusion

Reply:
Ah good point. I had suggested that it be a placeholder like `REPLACE_ME_DB`, but didn't validate that it actually changed to be that. Agree that this should be changed to a placeholder in cleanup.

Reply:
@shankari @JGreenlee It was changed to a placeholder but due to running the script, it changed once more (basically everytime you run the script the db location changes)

We could fix this in 2 ways I believe:

1. .gitignore the yaml from now on
2. Modify the script to revert the modification after launching compose?

Reply:
I would suggest `.gitignore` and add a section to the `README` highlighting that if anybody does change it, they should use `git add -f <filename>`

That optimizes for the common case (people using the docker-compose) and drops the burden of doing extra work on the uncommon case (people who are modifying it) and who are also more likely to understand and notice it.

Reply:
@shankari would you prefer a new pull request or directly change the repo since this PR was already merged?

Reply:
@TeachMeTW you can't directly change the repo; you don't have write access to it. It has to be a PR 😄

Reply:
@shankari moved to #123